### ISSUE TITLE
Unidentified Byte

### ISSUE THREAD

**Issue by beauwilliams** [pwntester/octo.nvim#156]
## Issue Description

I'm currently unable to run any of the Octo commands (I have tested)

Type: *bug report*

### Describe what happened (or what feature you want)

Execute any :Octo command

e.g the error output from :Octo pr list

```
Error executing vim.schedule lua callback: Vim:E474: Unidentified byte: ^[[1;38m{^[[m
  ^[[1;34m"data"^[[m^[[1;38m:^[[m ^[[1;38m{^[[m
    ^[[1;34m"repository"^[[m^[[1;38m:^[[m ^[[1;38m{^[[m
      ^[[1;34m"pullRequests"^[[m^[[1;38m:^[[m ^[[1;38m{^[[m
        ^[[1;34m"nodes"^[[m^[[1;38m:^[[m ^[[1;38m[^[[m^[[1;38m]^[[m^[[1;38m,^[[m
        ^[[1;34m"pageInfo"^[[m^[[1;38m:^[[m ^[[1;38m{^[[m
          ^[[1;34m"hasNextPage"^[[m^[[1;38m:^[[m ^[[33mfalse^[[m^[[1;38m,^[[m
          ^[[1;34m"endCursor"^[[m^[[1;38m:^[[m ^[[36mnull^[[m
        ^[[1;38m}^[[m
      ^[[1;38m}^[[m
    ^[[1;38m}^[[m
  ^[[1;38m}^[[m
^[[1;38m}^[[m
```

### How to reproduce it (as minimally and precisely as possible)

1. be in a git repo on neovim 0.5 release version
2. run command e.g :Octo list repo or :Octo list pr :Octo search search
3. get error `Error executing vim.schedule lua callback: Vim:E474: Unidentified byte:`

### Tell us your environment
MACOS 10.14.5
NVIM v0.5.0
Build type: Release
LuaJIT 2.1.0-beta3
gh version 1.12.1 (2021-07-01)
telescope & webdevicons installed latest

**Comment by pwntester** [comment]
Thanks for the report, I will try to investigate but it seems like, for whatever reason, the ansi codes returned by `gh` are messing things up for you. Will try to get them stripped out by setting the `NO_COLOR=1` env var

**Comment by pwntester** [comment]
Seems like a [bug in Plenary](https://github.com/nvim-lua/plenary.nvim/issues/152) prevents from using env variables. Will follow up with it

**Comment by pwntester** [comment]
This should be fixed now in https://github.com/pwntester/octo.nvim/pull/160 can you please give it a try and let me know if it works for you?

**Comment by pwntester** [comment]
implemented in #160
